README.md

```diff
@@ -180,14 +180,66 @@ pipeline {
 
 </details>
 
+<details>
+  <summary>Docker</summary>
+
+#### Preconditions 
+  1. Populate 'DOCKER_REG_URL' with the Artifactory Docker registry, for example - 'acme.jfrog.io'.
+  2. To build the Docker image, install the "Docker Pipeline" for Jenkins.
```

Comment:
... install the "Docker Pipeline" for Jenkins.... --> ... install the "Docker Pipeline" plugin on Jenkins....